rllib/core/models/base.py

```diff
@@ -17,45 +17,6 @@
 CRITIC: str = "critic"
 
 
-@ExperimentalAPI
```

Comment:
Moved this to the `configs.py` file for clarity and cleanliness. Compare: algorithm_config.py and algorithm.py